editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -1194,40 +1249,6 @@ void Polygon2DEditor::_uv_draw() {
 		//draw paint circle
 		uv_edit_draw->draw_circle(bone_paint_pos, bone_paint_radius->get_value() * EDSCALE, Color(1, 1, 1, 0.1));
 	}
-
-	rect.position = -uv_edit_draw->get_size();
```

Comment:
Simplified and moved this scroll updating logic to `_update_zoom_and_pan`, since there's no reason to update it on every redraw, only when zoom gets changed. Also helps solving problem of setting scroll position to value which gets clamped by scrollbar, then updating scrollbar limits.
Logic for hiding scrollbars isn't needed, even in this original implementation it never triggers.